extensions/sql-migration/src/api/sqlUtils.ts

```diff
@@ -84,14 +84,14 @@ function getSqlDbConnectionProfile(
 		databaseName: databaseName,
 		userName: userName,
 		password: password,
-		authenticationType: azdata.connection.AuthenticationType.SqlLogin,
+		authenticationType: 'SqlLogin',
```

Comment:
perhaps add a TODO comment to all these references indicating the enum should be added back in a future iteration?